pkgs/applications/misc/deckmaster/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, buildGoModule
+, fetchFromGitHub
+, makeWrapper
+, roboto
+}:
+
+buildGoModule rec {
+  pname = "deckmaster";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "muesli";
+    repo = "deckmaster";
+    rev = "v${version}";
+    sha256 = "sha256-q2rUHfAvTGXBAGrZUtHMuZr6fYWmpha+al2FG8sCC0Y=";
+  };
+
+  vendorSha256 = "sha256-kj4lRHuQ9e0TOC4p4Ak3AB3Lx0JN1jqXaVKlee9EtCg=";
+
+  proxyVendor = true;
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  buildInputs = [
+    roboto
```

Comment:
If the font is only used in the wrapper this is a noop.

Reply:
Wow, you learn something every day, didn't realise usage in wrapper would ensure a package was made available at run time!

Thanks @SuperSandro2000, I've removed the redundant `buildInputs`, built, tested, and force pushed.